chromadb/utils/embedding_functions.py

```diff
@@ -33,13 +34,16 @@ def __init__(self, api_key: str, model_name: str = "text-embedding-ada-002"):
     def __call__(self, texts: Documents) -> Embeddings:
         # replace newlines, which can negatively affect performance.
         texts = [t.replace("\n", " ") for t in texts]
-        # Call the OpenAI Embedding API in parallel for each document
+
+        # Call the OpenAI Embedding API
+        embeddings = self._client.create(input=texts, engine=self._model_name)["data"]
+
+        # Sort resulting embeddings by index
+        sorted_embeddings = sorted(embeddings, key=operator.attrgetter('index'))
```

Comment:
key=lambda embedding: embeddings['index'] works?

Reply:
Yes that should work fine too, I have a personal preference for attrgetter which is why I used that, but either should work fine